source/features/show-names.tsx

```diff
@@ -74,7 +73,8 @@ void features.add(__filebasename, {
 	additionalListeners: [
 		onNewsfeedLoad,
 	],
-	init: onetime(init),
+	onlyAdditionalListeners: true,
```

Comment:
Seems the events on the dashboard are always loaded asynchronously now.

Reply:
Yeah maybe we can skip this change for now. Maybe add `// TODO: Use onlyAdditionalListeners in February 2022 #4876 `